docs/spelling_wordlist.txt

```diff
@@ -310,6 +310,7 @@ Reinitialising
 Riccomini
 Roadmap
 Robinhood
+sigterm
```

Comment:
Make it `SIGTERM` then we don't need to add a new word to the spelling list? ;)

Reply:
No, just in the doc string where you say "send sigterm" change it to "send SIGTERM" -  spell checking doesn't look at function names